Tidied up (#487)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/whatsnew.md

# What's new in alpha?

> If you’ve used the GDK in pre-alpha  you may be interested in the latest changes.

Reviewer: ```suggestion
If you’ve used the GDK in pre-alpha you may be interested in the latest changes.
```

Author: Done, ta. Added comma, which is better IMHumbleO.

> If you’ve used the GDK in pre-alpha  you may be interested in the latest changes.

Reviewer 2: ```suggestion
If you’ve used the GDK in pre-alpha you may be interested in the latest changes.
```

> If you’ve used the GDK in pre-alpha  you may be interested in the latest changes.

Reviewer 3: ```suggestion
If you’ve used the GDK in pre-alpha you may be interested in the latest changes.
```

## Iterative improvements
- Upgraded the SpatialOS Unreal engine fork to support Unreal Engine 4.20.3.
- Upgraded to the new SpatialOS load balancer, providing fast multi-[worker]({{urlRoot}}/content/glossary#workers) iteration.

> - Removed the incremental build-time overhead to using the GDK by implementing [Dynamic Typebindings]({{urlRoot}}/content/glossary#dynamic-typebindings). This means you don't have to generate certain interop code. 

Reviewer 2: ```suggestion
- Removed the incremental build-time overhead of using the GDK by implementing [Dynamic Typebindings]({{urlRoot}}/content/glossary#dynamic-typebindings). This removes the additional code-generation and compile step.
```

Author: "The overhead to using" is better English. There is an overhead to doing something. I can change it if it makes everyone happier, though.

Reviewer 2: I took me a few reads to understand what the sentence meant. But I grew up in Australia.

For example: 

    ```
    "%UNREAL_HOME%\Engine\Binaries\Win64\UE4Editor-Win64-Debug.exe" "%~dp0%PROJECT_PATH%\%GAME_NAME%.uproject" 127.0.0.1
    ```

- The GDK now saves [startup Actors]({{urlRoot}}/content/generating-a-snapshot#startup-actors) into the [snapshot]({{urlRoot}}/content/generating-a-snapshot).
- Various stability fixes.

## New features
- [Authority]({{urlRoot}}/content/authority) is now based on SpatialOS authority.
- There is easy switching between native-Unreal networking and SpatialOS networking with a [toolbar checkbox]({{urlRoot}}/content/toolbars#switching-between-native-unreal-networking-and-spatialos-networking).
- The GDK is an Unreal plugin, so you don't need to symlink `UnrealGDK` directories.
- Improved [component]({{urlRoot}}/content/glossary#spatialos-component) replication support allows you to attach multiple components of the same type to an Actor (or "[entity]({{urlRoot}}/content/glossary#spatialos-entity)" in SpatialOS).
- [Cross-server RPCs]({{urlRoot}}/content/cross-server-rpcs) facilitate communication with Actors for which another server is [authoritative]({{urlRoot}}/content/authority).

> - `ClientTravel` allows you to move clients between [deployments]({{urlRoot}}/content/glossary#deployment), regardless of whether these deployments are local or in the cloud.

Reviewer 3: For ClientTravel and ServerTravel we should link to the Map Travel doc.
